src/main/resources/com/sonyericsson/jenkins/plugins/bfa/PluginImpl/config.jelly

```diff
@@ -32,6 +32,10 @@
                  description="${%Whether to display graphs or not. NOTE: The selected storage type must support this!}">
             <f:checkbox name="graphsEnabled" checked="${it.graphsEnabled}" default="true"/>
         </f:entry>
+        <f:entry title="${%Enable text when no failure causes are found}"
+                 description="${%Whether to display the no problems identified message at all}">
+            <f:checkbox name="noCausesEnabled" checked="${it.noCausesEnabled}" default="true"/>
+        </f:entry>
         <f:entry title="${%Text when no failure causes are found}"
```

Comment:
Would be better ux if the text selection was an optional property. I.e. when you click the enable text checkbox the text to put in appears.

Reply:
Good idea! but I'd like to keep the checkbox enabled by default (so the no indication text will appear), what do you think?

Reply:
Yes it should definitely be enabled by default. Especially for users upgrading (so try to test that just to be sure)